### ISSUE TITLE
About performance in Cub-200 finetune.

### ISSUE THREAD

**Issue by narrowsnap** [facebookresearch/WSL-Images#4]
Thank you for opensource this pretrained models.
I finetuned 32x32, 32x16 and 32x8 on Cub-200 dataset with 2 1080Ti, which only achive 87.5% accuracy on testset with image size 448x448. Image size with 224x224 only get 84.43% test acc. I find biger batch size got better performance but I only have 2 GPUs.
In the paper with 32x16 pretrained model reached 89.2 accuracy. 
Could you please show more details about finetune.
Thanks.

**Comment by dkm2110** [comment]
@narrowsnap what is the batch size per gpu u r using for the models? Since this is model trained on 940M images and 1.5k labels, you should expect an accuracy of about 87.9% or similar. Also for cUB it is important to do param sweep on LR, weight decay  and LR schedule.

**Comment by narrowsnap** [comment]
@dkm2110 
This is my experiments.

|model | img size | batch size/gpu | test acc |
|:--: |:--: |:--: |:--:|
|32x8 |448x448 |8 | 0.8702(0.8752 with mixup) |
|32x8 |224x224 |32 | - |
|32x16 |448x448 |4 | 0.8626 |
|32x16 |224x224 |16 | 0.8443 |

With SGD(model.parameters(), lr=0.001, momentum=0.9, weight_decay=1e-5) 
and MultiStepLR(optimizer, milestones=[30, 60], gamma=0.1)
epoches= 100.
